src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceAboutToSubmitHandler.java

```diff
@@ -428,10 +428,15 @@ private boolean isRepAddressInvalid(SscsCaseData caseData) {
     }
 
     private boolean isAddressInvalid(Address address) {
-        return null == address
-            || address.isAddressEmpty()
+        if (null == address) {
+            return true;
+        }
+
+        var isMainlandUkAndMissingPostcode = !YesNo.NO.equals(address.getInMainlandUk()) && isBlank(address.getPostcode());
```

Comment:
is inMainlandUK question optional? because if it is. we're not always going to get the No if they don't answer it

Reply:
yeah, it's crafted carefully, so that NULL or Yes give the desired behaviour